com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/BundleHelper.java

```diff
@@ -155,7 +155,7 @@ static public String formatResource(Map<String, Map<String, Object>> propertiesM
             return "";
         }
         String s = new String(data);
-        Template template = Mustache.compiler().compile(s);
+        Template template = Mustache.compiler().emptyStringIsFalse(true).compile(s);
```

Comment:
Please remember to comment on your PRs
What does this do? What was the problem? etc